app/src/main/java/com/onewelcome/showcaseapp/feature/info/InfoScreen.kt

```diff
@@ -3,23 +3,35 @@ package com.onewelcome.showcaseapp.feature.info
 import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material.icons.filled.Close
 import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
 import androidx.compose.material3.TopAppBar
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.hilt.navigation.compose.hiltViewModel
 import com.onewelcome.core.components.ShowcaseStatusCard
+import com.onewelcome.core.components.ShowcaseTableStatusCard
+import com.onewelcome.core.entity.MobileAuthEnrollmentStatus
 import com.onewelcome.core.theme.Dimensions
 import com.onewelcome.core.theme.separateItemsWithComa
+import com.onewelcome.core.theme.success
 import com.onewelcome.showcaseapp.R
 import com.onewelcome.showcaseapp.feature.info.InfoViewModel.State
 
+private val MOBILE_AUTH_TABLE_COLUMN_WIDTHS = listOf(0.4f, 0.3f, 0.3f)
```

Comment:
Do we need this as a parameter? Can't we hardcode it in the `ShowcaseTableCardScope`?